docs/how-to-upload-artifact.md

```diff
@@ -2,7 +2,7 @@
 
 This document shows how one can push an operator artifact onto `quay.io` or any other compatible app registry server. All commands in this document should be run in the root of this repository.
 
-## What is an Operator Artifact?
+## What is an Operator Artifact
```

Comment:
The `?` can stay.

Reply:
Okay, VS Code was complaining that headings shouldn't have trailing punctuation but I've added it back.